opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -74,6 +74,13 @@ namespace Opm {
         alternative_well_rate_init_ = EWOMS_GET_PARAM(TypeTag, bool, AlternativeWellRateInit);
     }
 
+    template<typename TypeTag>
+    BlackoilWellModel<TypeTag>::
+    BlackoilWellModel(Simulator& ebosSimulator) :
+        BlackoilWellModel(ebosSimulator, phaseUsageFromDeck(ebosSimulator_.vanguard().eclState()))
```

Comment:
Note: You can't use `ebosSimulator_` here since the data member is not initialised when we're here.  You'll need to use the `vanguard`from `ebosSimulator` (without the trailing underscore).

Reply:
Yes - thank you.